tests/gemm/test_mm_bf16_cutile.py

```
@@ -0,0 +1,107 @@
+import torch
+import torch.nn.functional as F
+
+from flashinfer import mm_bf16
+from flashinfer.utils import get_compute_capability
+
+
+def _cutile_available() -> bool:
+    """The cuTile path is optional — gracefully skip when the runtime is missing."""
+    try:
+        import cuda.tile  # noqa: F401
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)